docs/book/theme/css/custom.css

```diff
@@ -406,7 +406,12 @@ cite.literate-source > a::before {
 .tabset > input:nth-child(33):checked ~ .tab-panels > .tab-panel:nth-child(17),
 .tabset > input:nth-child(35):checked ~ .tab-panels > .tab-panel:nth-child(18),
 .tabset > input:nth-child(37):checked ~ .tab-panels > .tab-panel:nth-child(19),
```

Comment:
Thx for debugging this.

Context for others. Because of this css our tabs are limited to 25 (before this PR 20)